updates/24/updates.sql

```diff
@@ -462,6 +462,132 @@ INSERT INTO assets(identifier, name, type) VALUES("eip155:1/erc20:0x79F05c263055
+UPDATE common_asset_details SET cryptocompare="USDY" WHERE identifier="eip155:1/erc20:0x96F6eF951840721AdBF46Ac996b59E0235CB985C";
+INSERT INTO assets(identifier, name, type) VALUES("eip155:1/erc20:0x96F6eF951840721AdBF46Ac996b59E0235CB985C", "Ondo U.S. Dollar Yield", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:1/erc20:0x96F6eF951840721AdBF46Ac996b59E0235CB985C", "A", 1, "0x96F6eF951840721AdBF46Ac996b59E0235CB985C", 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:1/erc20:0x96F6eF951840721AdBF46Ac996b59E0235CB985C", "USDY", "ondo-us-dollar-yield", "USDY", NULL, NULL, NULL);
+UPDATE common_asset_details SET cryptocompare="SHDW" WHERE identifier="SHDW";
+INSERT INTO assets(identifier, name, type) VALUES("SHDW", "Shadow Token", "Y"); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("SHDW", "SHDW", "genesysgo-shadow", "SHDW", NULL, 1682143440, NULL);
+UPDATE common_asset_details SET cryptocompare="KLEVA" WHERE identifier="KLEVA";
+INSERT INTO assets(identifier, name, type) VALUES("KLEVA", "Kleva", "W"); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("KLEVA", "KLEVA", "kleva", "KLEVA", NULL, 1663053708, NULL);
+UPDATE common_asset_details SET cryptocompare="USDT", coingecko="USDT" WHERE identifier="eip155:8453/erc20:0x80e24fd799f47576a9c6e5Bf69c40c9d99146491";
+INSERT INTO assets(identifier, name, type) VALUES("eip155:8453/erc20:0x80e24fd799f47576a9c6e5Bf69c40c9d99146491", "USDT Token", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:8453/erc20:0x80e24fd799f47576a9c6e5Bf69c40c9d99146491", "A", 8453, "0x80e24fd799f47576a9c6e5Bf69c40c9d99146491", 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:8453/erc20:0x80e24fd799f47576a9c6e5Bf69c40c9d99146491", "USDT", "tether", "USDT", NULL, NULL, NULL);
+UPDATE common_asset_details SET cryptocompare="LUSD" WHERE identifier="eip155:8453/erc20:0x368181499736d0c0CC614DBB145E2EC1AC86b8c6";
+INSERT INTO assets(identifier, name, type) VALUES("eip155:8453/erc20:0x368181499736d0c0CC614DBB145E2EC1AC86b8c6", "LUSD Stablecoin", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:8453/erc20:0x368181499736d0c0CC614DBB145E2EC1AC86b8c6", "A", 8453, "0x368181499736d0c0CC614DBB145E2EC1AC86b8c6", 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:8453/erc20:0x368181499736d0c0CC614DBB145E2EC1AC86b8c6", "LUSD", "liquity-usd", "LUSD", NULL, NULL, NULL);
```

Comment:
timestamps are missing in some assets like this one. I've sent the script patch in discord which may help.